### PR TITLE
Change number type to text type

### PR DESCRIPTION
**Pull request by mespeche**
It's impossible to set a decimal digit with the number type.

### CODE REVIEW

**Comment by lovenunu** [comment]
It is, using ```step=0.01``` attribute. In fact, only this attr is missing :)

**Comment by mespeche** [comment]
Yes it's true, it's done ;-)

File: AdminIncludes/module_configuration.html [hunk 1]
```diff
@@ -262,7 +262,7 @@
                                 <input type=\"hidden\" name=\"area\" value=\"{$ID}\"/>
                                 <input type=\"hidden\" name=\"weight\" value=\"{$MAX_WEIGHT}\"/>
                                 <label for=\"price_edit_{$ID}_{$MAX_WEIGHT}\">{intl l='Price (€)' d='socolissimo.ai'}</label>
-                                <input type=\"number\" id=\"price_edit_{$ID}_{$MAX_WEIGHT}\" class=\"form-control\" name=\"price\" value=\"{$PRICE}\" pattern=\"\\d+\\.?\\d*\" required/>"
+                                <input type=\"text\" id=\"price_edit_{$ID}_{$MAX_WEIGHT}\" class=\"form-control\" name=\"price\" value=\"{$PRICE}\" pattern=\"\\d+\\.?\\d*\" required/>"
```
Comment (lovenunu) [inline AdminIncludes/module_configuration.html]: Could you change this line too please ? That'd be great !

**Comment by mespeche** [comment]
Done ;-)

**Pull request merged**